<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mdiJobDetails</class>
 <widget class="QWidget" name="mdiJobDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>919</width>
    <height>412</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" colspan="4">
    <widget class="QTableView" name="tableView">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>60</number>
     </attribute>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblFilter">
     <property name="text">
      <string>Filter by:</string>
     </property>
     <property name="buddy">
      <cstring>editFilterBy</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QLineEdit" name="editFilterBy"/>
   </item>
   <item row="2" column="0">
    <widget class="QSplitter" name="splitter_5">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Job reference:</string>
      </property>
      <property name="buddy">
       <cstring>editReference</cstring>
      </property>
     </widget>
     <widget class="QLineEdit" name="editReference"/>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Owner of the job</string>
      </property>
      <property name="buddy">
       <cstring>editOwner</cstring>
      </property>
     </widget>
     <widget class="QLineEdit" name="editOwner"/>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Site manager:</string>
      </property>
      <property name="buddy">
       <cstring>editSiteManager</cstring>
      </property>
     </widget>
     <widget class="QLineEdit" name="editSiteManager"/>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSplitter" name="splitter_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Postcode:</string>
      </property>
      <property name="buddy">
       <cstring>editPostcode</cstring>
      </property>
     </widget>
     <widget class="QLineEdit" name="editPostcode"/>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Address:</string>
      </property>
      <property name="buddy">
       <cstring>editAddress</cstring>
      </property>
     </widget>
     <widget class="QTextEdit" name="editAddress"/>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QSplitter" name="splitter_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Job start:</string>
      </property>
      <property name="buddy">
       <cstring>editJobStart</cstring>
      </property>
     </widget>
     <widget class="QDateEdit" name="editJobStart">
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>To finish job by:</string>
      </property>
      <property name="buddy">
       <cstring>editJobFinishBy</cstring>
      </property>
     </widget>
     <widget class="QDateEdit" name="editJobFinishBy">
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Job finished:</string>
      </property>
      <property name="buddy">
       <cstring>editJobFinished</cstring>
      </property>
     </widget>
     <widget class="QDateEdit" name="editJobFinished">
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QSplitter" name="splitter_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Job Value:</string>
      </property>
      <property name="buddy">
       <cstring>editJobValue</cstring>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="editJobValue">
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="showGroupSeparator" stdset="0">
       <bool>true</bool>
      </property>
      <property name="prefix">
       <string>£ </string>
      </property>
      <property name="maximum">
       <double>100000000000000000000.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Job spendings:</string>
      </property>
      <property name="buddy">
       <cstring>editJobSpendings</cstring>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="editJobSpendings">
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="showGroupSeparator" stdset="0">
       <bool>true</bool>
      </property>
      <property name="prefix">
       <string>£ </string>
      </property>
      <property name="maximum">
       <double>100000000000000000000.000000000000000</double>
      </property>
     </widget>
     <widget class="QCheckBox" name="editJobClosed">
      <property name="text">
       <string>Job finished</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QPushButton" name="buttonClose">
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>editFilterBy</tabstop>
  <tabstop>editReference</tabstop>
  <tabstop>editOwner</tabstop>
  <tabstop>editSiteManager</tabstop>
  <tabstop>editPostcode</tabstop>
  <tabstop>editAddress</tabstop>
  <tabstop>editJobValue</tabstop>
  <tabstop>editJobStart</tabstop>
  <tabstop>editJobFinishBy</tabstop>
  <tabstop>editJobFinished</tabstop>
  <tabstop>editJobClosed</tabstop>
  <tabstop>tableView</tabstop>
  <tabstop>buttonClose</tabstop>
  <tabstop>editJobSpendings</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
